generators/client/templates/angular/src/main/webapp/app/core/_core.module.ts

```diff
@@ -0,0 +1,83 @@
+import { NgModule, LOCALE_ID } from '@angular/core';
+import { DatePipe, registerLocaleData } from '@angular/common';
+import { Title } from '@angular/platform-browser';
+<%_ const localeId = getLocaleId(nativeLanguage); _%>
+import locale from '@angular/common/locales/<%= localeId %>';
+
+import {
+    <%_ if (websocket === 'spring-websocket') { _%>
+    <%=jhiPrefixCapitalized%>TrackerService,
+    WindowRef,
+    <%_ } _%>
+    <%_ if (enableTranslation) { _%>
+    JhiLanguageHelper,
+    <%_ } _%>
+    LoginService,
+    <%_ if (authenticationType !== 'oauth2') { _%>
+    LoginModalService,
+    <%_ } _%>
+    <%_ if (enableSocialSignIn) { _%>
+    SocialService,
+    <%_ } _%>
+    AccountService,
+    StateStorageService,
+    Principal,
+    CSRFService,
+    AuthServerProvider,
+    <%_ if (!skipUserManagement || authenticationType === 'oauth2') { _%>
+    UserService,
+    <%_ } _%>
+    UserRouteAccessService
+} from './';
+<%_ if (enableI18nRTL) { _%>
+import {
+    FindLanguageFromKeyPipe
+} from '../shared';<%_ } _%>
```

Comment:
if something from shared is needed here shouldnt it be moved here as well?

Reply:
I let this pipe in the shared because it's a pipe and can be used as a classic pipe in the HTML (so we need it in the declarations/exports arrays) or as a service (in the providers array), so it has to stay in shared...
The CoreModule is imported once in the AppModule and should only contains providers.

Reply:
You are right. Ok

Reply:
app/shared plz